iina/SleepPreventer.swift

```diff
@@ -15,8 +15,44 @@ class SleepPreventer: NSObject {
 
   static private var assertionID = IOPMAssertionID()
 
+  static private var haveShownAlert = false
+
   static private var preventedSleep = false
 
+  /// Ask macOS to not dim or sleep the display.
```

Comment:
A brief description is fine but I'd strongly recommend just linking to the issues themselves rather than copying their contents here, because they'll remain up-to-date and are a better place to put extended discussion

Reply:
That came from my bias for keeping information with the code after encountering broken links on work projects. I've changed that portion of the comment to just refer to the two issues.

Reply:
We are not likely to migrate to JIRA anytime soon :P